chronix-server-test-integration/src/main/java/de/qaware/chronix/solr/CSVImporter.java

```diff
@@ -52,15 +51,7 @@
     public static final List<Long> LIST_LONG_FIELD = Lists.newArrayList(11L, 25L);
     public static final List<Double> LIST_DOUBLE_FIELD = Lists.newArrayList(1.5D, 2.6D);
 
-    public static byte[] bytes;
-
-    static {
-        try {
-            bytes = "String as byte".getBytes("UTF-8");
-        } catch (UnsupportedEncodingException e) {
-            //Does not happen
-        }
-    }
+    public static byte[] BYTES = "String as byte".getBytes();
```

Comment:
\[Checkstyle\] ERROR: Variable 'BYTES' must be private and have accessor methods\.